src/components/Account/Menu.tsx

```diff
@@ -50,11 +50,22 @@ const AccountMenu: React.FC<BoxProps> = (props) => {
             <Icon as={UserSquare} />
             <Trans i18nKey='profile.title'>Profile</Trans>
           </MenuItem>
+          <MenuItem as={RouterLink} to='https://developer.vocdoni.io/' target='_blank'>
+            <Icon as={Paperclip} />
+            <Trans i18nKey='menu.documentation'>Documentation</Trans>
+          </MenuItem>
           <MenuDivider />
           <MenuItem onClick={logout}>
             <Icon as={LogOut01} />
             <Trans i18nKey='logout'>Logout</Trans>
           </MenuItem>
+          <MenuDivider />
+          <MenuItem fontSize={'xs'} as={RouterLink} to={Routes.terms} target='_blank'>
+            <Trans i18nKey='menu.terms'>Terms</Trans>
+          </MenuItem>
+          <MenuItem fontSize={'xs'} as={RouterLink} to={Routes.privacy} target='_blank'>
+            <Trans i18nKey='menu.privacy'>Privacy</Trans>
+          </MenuItem>
```

Comment:
Should not be `target='_blank'`. Also, you should be using `isExternal` instead of target _blank, but here is not required at all.